c/array-industry-pattern/array_of_struct_ptr_cond_init_true-unreach-call.i

```diff
@@ -1,22 +1,31 @@
+
 typedef unsigned int size_t;
 extern void *malloc (size_t __size) __attribute__ ((__nothrow__ , __leaf__)) __attribute__ ((__malloc__)) ;
 extern void __VERIFIER_error() __attribute__ ((__noreturn__));
+extern int __VERIFIER_nondet_int(void);
 void __VERIFIER_assert(int cond) { if(!(cond)) { ERROR: __VERIFIER_error(); } }
+
 struct _S
 {
  int *p;
  int n;
 };
 typedef struct _S S;
+
 S *a[100000];
-extern int user_read();
+int user_read();
+
 int main()
 {
  int i;
+
```

Comment:
There is no need for these new lines and they are distracting from the main part of this commit.